tensorboard/backend/event_processing/plugin_event_accumulator.py

```diff
@@ -278,6 +278,16 @@ def PluginTagToContent(self, plugin_name):
                     # before setting `released_metalock`.
                     pass
 
+    def ActivePlugins(self):
+        """Return a list of plugins with summary data.
```

Comment:
Any reason not to return a frozenset instead?  That way it's the same return type as on EventMultiplexer.